Common/Data/Format/IniFile.cpp

```diff
@@ -29,6 +29,24 @@
 
 #include "Common/StringUtils.h"
 
+int strvcasecmp(const std::string_view &lhs, const std::string_view &rhs) {
+	int zret = 0;
+	size_t n = rhs.size();
+
+	// Seems a bit ugly but size comparisons must be done anyway to get the @c strncasecmp args.
```

Comment:
Hm, well, since we only care if they equal it feels like we could ignore the +/- of it.  Might be quicker to skip any byte compare when they're known different lengths.

-[Unknown]

Reply:
Yeah true that, I just grabbed this utility function somewhere.